flasksrc/session.py

```diff
@@ -0,0 +1,133 @@
+from marshmallow import Schema, fields, ValidationError
+from flask_restful import Resource, abort, reqparse
+from flask_apispec import marshal_with
+from flask_apispec.views import MethodResource
+from flask import request
+from flasksrc.db import get_db
+
+
+# Schema definitions
+class ToyNetSessionPostReq(Schema):
+    toynet_topo_id = fields.Int(required=True)
+    toynet_user_id = fields.Str(required=True)
+
+
+class ToyNetSessionPostResp(Schema):
+    status = fields.Bool(required=True)
+    toynet_session_id = fields.Int(required=True)
+
+
+class ToyNetSessionByIdGetResp(Schema):
+    topo_id = fields.Int(required=True)
+    topology = fields.Str(required=True)
+    user_id = fields.Str(required=True)
+
+
+class ToyNetSession(MethodResource, Resource):
+    @marshal_with(ToyNetSessionPostResp)
+    def post(self):
+        try:
+            req = ToyNetSessionPostReq().load(request.form)
+        except ValidationError as e:
+            abort(400, message=f'malformed request: {e.messages}')
+
+        toynet_topo_id = req['toynet_topo_id']
+        toynet_user_id = req['toynet_user_id']
+
+        db = get_db()
+
+        try:
+            topo_rows = db.execute(
+                'SELECT topology' +
+                ' FROM toynet_topos' +
+                ' WHERE topo_id = (?)',
+                (str(toynet_topo_id),)
+            ).fetchall()
+        except Exception as e:
+            print(e.args[0])
+            abort(500, message='topo_id query failed: {}'.format(toynet_topo_id))
+
+        if not len(topo_rows):
+            abort(400, message='topo_id is invalid: {}'.format(toynet_topo_id))
+
+        try:
+            user_rows = db.execute(
+                'SELECT username' +
+                ' FROM users' +
+                ' WHERE username = (?)',
+                (str(toynet_user_id),)
+            ).fetchall()
+        except Exception as e:
+            print(e.args[0])
+            abort(500, message='user_id query failed: {}'.format(toynet_user_id))
+
+        if not len(user_rows):
+            abort(400, message='user_id is invalid: {}'.format(toynet_user_id))
+
+        try:
+            cur = db.cursor()
+            cur.execute(
+                    'INSERT INTO toynet_sessions(topo_id, topology, user_id)' +
```

Comment:
Remove all `+` to align with:
https://github.com/Project-Reclass/toynet-flask/blob/main/flasksrc/value.py#L54

Reply:
remove 1 level of indentation